en-US/firefox/whatsnew_73.lang

```diff
@@ -0,0 +1,70 @@
+## NOTE: Demo page at https://www-demo5.allizom.org/en-US/firefox/73.0/whatsnew/all/
```

Comment:
- Remove en-US in the link,
- Make sure to include this in note as well: https://www-dev.allizom.org/%LOCALE%/firefox/73.0/whatsnew/all/